src/Microsoft.Azure.WebJobs.Host/Hosting/WebJobsHostBuilderExtensions.cs

```diff
@@ -137,16 +139,32 @@ public static IHostBuilder ConfigureWebJobs(this IHostBuilder builder, Action<Ho
         }
 
         /// <summary>
-        /// Applies scale configuration to the specified <see cref="IHostBuilder"/>.
+        /// Configures the specified <see cref="IHostBuilder"/> as a scale manager host.
         /// </summary>
         /// <param name="builder">The <see cref="IHostBuilder"/> to configure.</param>
+        /// <param name="configureScaleOptions">Configuration action for <see cref="ScaleOptions"/>.</param>
         /// <returns>The <see cref="IHostBuilder"/>.</returns>
-        public static IHostBuilder AddScale(this IHostBuilder builder)
+        public static IHostBuilder ConfigureWebJobsScale<TConcurrencyStatusRepository, TScaleMetricsRepository>(this IHostBuilder builder, Action<ScaleOptions> configureScaleOptions, string hostId)
```

Comment:
Required repository impls. provided as type params. HostID also specified, and using existing fixed ID provider